app/controllers/api/v1/admin/users_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Api::V1::Admin::UsersController < Api::V1::BaseController
+  before_action :authenticate_user!
+  before_action :authorize_user
+  before_action :load_user, except: [:index, :roles, :statuses]
+  after_action :verify_authorized
+
+  def index
+    @users = User.all
```

Comment:
we need to add pagination here. If we have thousands of users, there can be a problem with handling all this (because of memory)